### PR TITLE
Update dev setup instructions

### PR DESCRIPTION
**Pull request by soyacz**
I've managed to setup my dev environment and organized steps to do that in a convinient way.

### CODE REVIEW

**Review by k0machi — COMMENTED** [review]
This should have `WEBPACK_ENVIRONMENT=development` for faster compilation times, otherwise compilation step will be a bit long

File: dev-setup.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+```bash
+python dev-db/import_data.py
+```
+Sporadical errors like `Test entity missing for key "enterprise-2023.1/upgrade-with-raft/rolling-upgrade-centos8-with-raft-test", run won't be visible until this is corrected`
+can be ignored.
+
+## Run the application
+Compile frontend files from `/frontend` into `/public/dist`. Add --watch to recompile files on change.
+```bash
+yarn webpack --watch
```
Comment (k0machi) [inline dev-setup.md:176]: Apologies, meant this

Comment (soyacz) [reply dev-setup.md:176]: like: `WEBPACK_ENVIRONMENT=development yarn webpack --watch`?

Comment (k0machi) [reply dev-setup.md:176]: Yes

Comment (soyacz) [reply dev-setup.md]: fixed. Also moved this dev-doc.md to docs folder

**Review by k0machi — APPROVED** [review]

**Pull request merged**